src/main/java/org/sonarlint/intellij/core/ProjectServerNotifications.java

```diff
@@ -46,7 +48,8 @@
 import static org.sonarlint.intellij.config.Settings.getSettingsFor;
 
 public class ProjectServerNotifications {
-  private static final NotificationGroup SONARQUBE_GROUP = NotificationGroup.balloonGroup("SonarLint: SonarQube Events");
+  private static final NotificationGroup SERVER_NOTIFICATIONS_GROUP =
+    new NotificationGroup("SonarLint: Server Notifications", NotificationDisplayType.STICKY_BALLOON, true, "SonarLint");
```

Comment:
Probably not a big deal but by changing this text we might lose some user preferences. Users can change settings for specific notifications.
![Screenshot from 2020-11-13 14-57-21](https://user-images.githubusercontent.com/64742703/99079812-baa11700-25c0-11eb-9767-b793d0df6103.png)

Reply:
I know, but I don't think it is worth trying to migrate old settings. Anyway with the refresh of the feature, it is maybe good to "reset" settings.